src/index.jsx

```diff
@@ -6,17 +6,17 @@ import { AppContainer } from 'react-hot-loader';
 import configureStore from 'configureStore.js';
 import createHistory from 'history/createBrowserHistory';
 import AppView from 'views/AppView.jsx';
-import 'font-awesome/scss/font-awesome.scss';
-import 'bulma/css/bulma.css';
 import 'styles.scss';
 
+
 const history = createHistory();
 
 const store = configureStore(history);
 const rootElement = document.getElementById('app');
 
+
 function renderApp(AppComponent) {
-  ReactDOM.render(
+  ReactDOM.hydrate(
```

Comment:
From React 16 and forward we no longer use render to start an app that is server side rendered, we have to call `ReactDOM.hydrate` that's lightweight, it will only attach events handlers to the app, rather then regenerating the HTML from the scratch